app/src/main/java/org/stepik/android/view/video_player/ui/activity/VideoPlayerActivity.kt

```diff
@@ -245,14 +236,13 @@ class VideoPlayerActivity : AppCompatActivity(), VideoPlayerView, VideoQualityDi
         playerView.showController()
     }
 
-    override fun setIsRotateVideo(isRotateVideo: Boolean) {
-        this.isRotateVideo = isRotateVideo
-        requestedOrientation =
-            if (isRotateVideo) {
-                ActivityInfo.SCREEN_ORIENTATION_FULL_SENSOR
-            } else {
-                ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED
-            }
+    override fun setIsLandscapeVideo(isLandScapeVideo: Boolean) {
+        this.isLandscapeVideo = isLandScapeVideo
+        if (isLandscapeVideo) {
+            playerView.exo_controller.exo_fullscreen_icon.setImageDrawable(AppCompatResources.getDrawable(this, R.drawable.ic_fullscreen_exit))
```

Comment:
`AppCompatImageView` + `setImageResource`, argument can be lifted from if statement